mqttwarn.py

```diff
@@ -340,7 +340,10 @@ def join(self):
     sys.exit(2)
 
 LOGLEVEL  = cf.loglevelnumber
-LOGFILE   = cf.logfile
+
+# if MQTTWARNLOG env variable exists, use it; otherwise, use config file
```

Comment:
Would you please the word "config" to "configured"? I think that's what you mean.

Reply:
Does this sound better? (I think it better explains it) If so, let me know and I'll push the change. Or, I can just change 'config' to 'configured', if you prefer that wording.

`# if MQTTWARNLOG env variable exists, use it; otherwise, use 'logfile' option from mqttwarn.ini`

Reply:
That's good.